cns/ipampool/metrics.go

```diff
@@ -84,7 +84,7 @@ var (
 	ipamPrimaryIPCount = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Name:        "cx_ipam_primary_ips",
-			Help:        "NC Primary IP count.",
+			Help:        "NC Primary IP count (unassignable).",
```

Comment:
As this help is going to be a tooltip and it is counting primary IPs across the cluster I would describe it as:
```suggestion
			Help:        "Cluster Primary IP count (unassignable to pods).",
```